spec/unit/validator_spec.rb

```diff
@@ -381,6 +381,16 @@ def podspec_path(name = 'JSONKit', version = '1.4')
         podfile.sources.should == %w(https://github.com/CocoaPods/Specs.git)
       end
 
+      it 'avoids creation of sources when source_urls is empty and no dependencies' do
```

Comment:
Hm okay. Maybe we should go back to the shortcutting solution if that's the most straightforward. I was under the impression this would be easier to bypass. What do you think?